lib/compat/wordpress-6.2/get-global-styles-and-settings.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * API to interact with global settings & styles.
+ *
+ * @package gutenberg
+ */
+
+if ( ! function_exists( 'wp_theme_has_theme_json' ) ) {
+	/**
+	 * Whether a theme or its parent have a theme.json file.
+	 *
+	 * @param boolean $clear_cache Whether the cache should be cleared and theme support recomputed. Default is false.
+	 *
+	 * @return boolean
+	 */
+	function wp_theme_has_theme_json( $clear_cache = false ) {
+		static $theme_has_support = null;
+
+		if ( false === $clear_cache ) {
```

Comment:
```suggestion
		if ( true === $clear_cache ) {
```

We should check against the `true` here unless I'm missing something.